lib/src/collect.dart

```diff
@@ -284,7 +284,7 @@ Future<List<Map<String, dynamic>>> _getCoverageJson(
 
   Future<void> processFunction(FuncRef funcRef) async {
     final func = await service.getObject(isolateRef.id!, funcRef.id!) as Func;
-    if (func.implicit ?? false) {
+    if ((func.implicit ?? false) || (func.isAbstract ?? false)) {
```

Comment:
Maybe? 🤷 

```suggestion
    if (func.implicit ?? func.isAbstract ?? false) {
```

Reply:
This would change the result if implicit is false but abstract is true. Old code would say true, but this change would say false.

Reply:
Ah! Yup...